util/update_lints.py

```diff
@@ -21,12 +21,18 @@
     " (?P<desc>(?:[^"\\]+|\\.)*) " \s* [})]
 ''', re.VERBOSE | re.DOTALL)
 
+declare_restriction_lint_re = re.compile(r'''
+    declare_restriction_lint! \s* [{(] \s*
+    pub \s+ (?P<name>[A-Z_][A-Z_0-9]*) \s*,\s*
+    " (?P<desc>(?:[^"\\]+|\\.)*) " \s* [})]
```

Comment:
It is 3AM, so the regex-parsing section of my brain has gone to sleep -- I'm not even going to try :stuck_out_tongue_closed_eyes: